src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
```

Comment:
move this to the 'base.scss' file to have only imports here. In the future, you can increase basic styling for elements and your main scss would stay clear.